assemble/bin/accumulo-cluster

```diff
@@ -304,16 +417,21 @@ function stop_here() {
   if grep -Eq 'localhost|127[.]0[.]0[.]1' "${conf}/tservers"; then
     ${accumulo_cmd} admin stop localhost
   else
+    IFS_ORIG=$IFS
+    IFS=","
     for host in "${hosts_to_check[@]}"; do
-      if grep -q "$host" "${conf}/tservers"; then
-        ${accumulo_cmd} admin stop "$host"
-      fi
+      for tserver in ${HOSTS["TSERVERS"]}; do
+        if grep -q "$host" "$tserver"; then
+          ${accumulo_cmd} admin stop "$host"
+        fi
+      done
     done
+    IFS=$IFS_ORIG
```

Comment:
This looks like the correct use of IFS... but messing with IFS can have unexpected consequences, especially when a future dev comes in later and changes something without understanding the implications.

Instead of having these comma-separated, just make them space separated or use an array to begin with, as in the outer loop. It'd be much more reliable.